art_show/site_sections/art_show_admin.py

```diff
@@ -70,6 +77,99 @@ def history(self, session, id):
                 .order_by(Tracking.when).all()
         }
 
+    @unrestricted
+    def bid_sheet_barcode_generator(self, data):
```

Comment:
After I wrote this function based off a similar function for check-in barcodes, I found it was much too slow to use with our bid sheet printing -- loading even a few barcodes slowed down page load dramatically. There wasn't any strong reason to remove it, though.